modules/ossm-directing-egress-traffic-through-a-gateway-kubernetes-gateway-api-ambient-mode.adoc

```diff
@@ -0,0 +1,142 @@
+// This procedure is used in the following assembly:
+// * service-mesh-docs-main/gateways/ossm-directing-outbound-traffic-through-a-gateway
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-directing-egress-traffic-through-gateway-kubernetes-gateway-api-ambient-mode_{context}"]
+= Directing egress traffic through a gateway using the {k8s} Gateway API in ambient mode
+
+[role="_abstract"]
+Use the {k8s} Gateway API and waypoint proxy to direct outbound HTTP traffic through an egress gateway.
+
+.Prerequisites
+
+* You have installed the {SMProduct} Operator version 3.2 or later.
+
+* You configured the `Istio` and `IstioCNI` resources with ambient profile.
+
+* You have created a `Ztunnel` resource.
+
+.Procedure
+
+. Optional: Enable the {k8} Gateway API custom resource definitions (CRDs). 
++
+[NOTE]
+====
+As of {k8s} 1.28 and {ocp-product-title} 4.18 or earlier version of {product-title}, the {k8s} Gateway API CRDs are not available by default and you must install the CRDs before you can use them. {ocp-product-title} 4.19 and later versions install the CRDs by default.
+====
+
+. Create a namespace called `egress-gateway` by running the following command:
++
+[source,terminal]
+----
+$ oc create namespace egress-gateway
+----
+
+. Apply the ambient mode label to the namespace by running the following command:
```

Comment:
Can we add the name of the label in this sentence, like you've done in the previous statement, to maintain consistency?